pkgs/by-name/wh/whatsie/package.nix

```
@@ -0,0 +1,58 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  libX11,
+  libxcb,
+  qt5,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "whatsie";
+  version = "4.15.3";
+
+  src = fetchFromGitHub {
+    owner = "keshavbhatt";
+    repo = "whatsie";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-fbbyh8Q1jbKY+4ZEexDvIrhdcP+uCYAnhtUQsSqS0jw=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/src";
+
+  buildInputs = [
+    libX11
+    libxcb
+    qt5.qtwebengine
+  ];
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    qt5.qmake
+  ];
+
+  strictDeps = false;
+
+  enableParallelBuilding = true;
+
+  preBuild = ''
+    export QT_WEBENGINE_ICU_DATA_DIR=${qt5.qtwebengine.out}/resources
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 whatsie -t $out/bin
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/keshavbhatt/whatsie";
+    description = "Feature rich WhatsApp Client for Desktop Linux";
+    license = lib.licenses.mit;
+    mainProgram = "whatsie";
+    maintainers = with lib.maintainers; [ ajgon ];
+    platforms = [ "x86_64-linux" ];
```

Is there anything besides "I don't have an ARM machine to test" preventing this to be
```suggestion
    platforms = lib.platforms.linux;
```

?

---

No :)